module/zfs/ddt.c

```diff
@@ -622,7 +622,7 @@ void
 ddt_init(void)
 {
 	ddt_cache = kmem_cache_create("ddt_cache",
-	    sizeof (ddt_t), 0, NULL, NULL, NULL, NULL, NULL, 0);
+	    sizeof (ddt_t), 0, NULL, NULL, NULL, NULL, NULL, KMC_KVMEM);
```

Comment:
These objects are relatively small (24864) so there's no need to request the `KMC_KVMEM` cache here.   Let's leave the flags set to `0` here.